AK/Random.h

```diff
@@ -17,7 +17,7 @@
 #    include <unistd.h>
 #endif
 
-#if defined(AK_OS_MACOS)
+#if defined(AK_OS_MACOS) || defined(AK_OS_WIN32)
 #    include <sys/random.h>
```

Comment:
so if the goal is to eventually support native win32 without cygwin why not make the investment now to use c++'s `#include <random>`  which already has cross OS support instead of using the posix random. instead of `getentropy` below maybe `std::random_device::entropy()`?